src/reactnativeapp/package.json

```
@@ -0,0 +1,76 @@
+{
+  "name": "reactnativeapp",
+  "main": "expo-router/entry",
+  "version": "1.0.0",
+  "scripts": {
+    "start": "ENVFILE=.env expo start",
```

Can we add a `start-ios` target here as well to make it easy to start the ios specific JS bundle when used with XCode?

```suggestion
    "start": "ENVFILE=.env expo start",
    "start-ios": "ENVFILE=.env expo start --ios",
```

---

this is actually a bit different of a way to run the app than the steps describe in the README, this launches a sandbox in the simulator called Expo Go which runs the JS bundle without building the app for iOS. It can be a quicker way to see things working but requires that the app not make use of any native code, I hadn't tried it for this app previously but looks like it fails due to some packages having native requirements

---

The instructions in the readme say to start a dev service by running this command: `npm run start`. When I run that command, I'm presented with a menu that looks like this:

```› Using Expo Go
› Press s │ switch to development build

› Press a │ open Android
› Press i │ open iOS simulator
› Press w │ open web

› Press j │ open debugger
› Press r │ reload app
› Press m │ toggle menu
› Press o │ open project code in your editor
```
From here I hit `i` which ends up opening the iOS simulator with an Expo app with some JS errors. I switch over to XCode and hit the Play button; it builds the app and then starts it in the Simulator.

I assume these are the intended original steps. If so, I would like us to bypass forcing the user to press `i` to Open the iOS simulator.

---

ah I see now, ya that can be ignored since hitting play through XCode will launch the simulator, I added a note about that here if that makes things clearer: https://github.com/open-telemetry/opentelemetry-demo/pull/1781/commits/9a190ce286e21c0926f054307eb41ab14b81d13a